aci/volumes.go

```diff
@@ -164,7 +165,47 @@ func errorEvent(resource string) progress.Event {
 	}
 }
 
+func checkVolumeUsage(ctx context.Context, aciContext store.AciContext, id string) error {
+	resourceGroups, err := NewACIResourceGroupHelper().ListGroups(ctx, aciContext.SubscriptionID)
+	if err != nil {
+		return err
+	}
+	for _, rg := range resourceGroups {
```

Comment:
You can probably avoid this loop, and use an ACI call that does not filter on resource group to get containers

Reply:
I tried this already. The problem is that from the list of `container groups` I cannot know from which `resource group` it comes from.